src/ios/HappieCameraViewController.swift

```diff
@@ -28,6 +30,33 @@ import AssetsLibrary
 
     var delegate:cameraDelegate?
 
+    var uMM: CMMotionManager!
+    
+    override func viewWillAppear( p: Bool ) {
+        super.viewWillAppear( p )
+        uMM = CMMotionManager()
+        uMM.accelerometerUpdateInterval = 0.2
+        uMM.startAccelerometerUpdatesToQueue( NSOperationQueue() ) { p, _ in
+            if p != nil {
+                var newRawOrient:String = "";
+                   newRawOrient = abs( p!.acceleration.y ) < abs( p!.acceleration.x )
+                        ?   p!.acceleration.x > 0 ? "Right"  :   "Left"
+                        :   p!.acceleration.y > 0 ? "Down"   :   "Up"
+                if self.rawOrient != newRawOrient {
+                    self.rawOrient = newRawOrient
+                    print(self.rawOrient)
```

Comment:
@rishikapadia1 lets remove the print unless you think it is necessary to keep for a while. Same with line 310